atlasdb-calcite/build.gradle

```diff
@@ -0,0 +1,55 @@
+apply from: "../gradle/publish-jars.gradle"
+apply from: "../gradle/shared.gradle"
+
+apply plugin: 'org.inferred.processors'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.6'
+    }
+}
+
+apply plugin: 'com.google.protobuf'
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:2.6.1'
+    }
+    generatedFilesBaseDir = "$projectDir/src"
+}
+
+apply plugin: 'application'
+mainClassName = 'com.palantir.atlasdb.calcite.SqlShell'
+applicationName = 'asql'
+
+repositories {
+    maven {
+        url "http://conjars.org/repo"
+    }
+}
+
+dependencies {
+    compile project(':atlasdb-dagger')
+    compile project(':atlasdb-service')
+    compile group: 'io.airlift', name: 'airline', version: '0.7'
+    compile(group: 'org.apache.calcite', name: 'calcite-core') {
+        exclude group: 'com.google.protobuf', module: 'protobuf-java'
+        exclude group: 'commons-lang', module: 'commons-lang'
+    }
+    compile group: "com.google.protobuf", name: "protobuf-java", version: libVersions.protobuf
+    compile group: "commons-lang", name: "commons-lang", version: libVersions.commons_lang
+    compile group: 'sqlline', name: 'sqlline', version: '1.1.9'
```

Comment:
curious why not [latest 1.2.0](https://github.com/julianhyde/sqlline/blob/master/HISTORY.md#120-2016-10-26) and should probably pull these versions out into the global version.props